backend/de.metas.business/src/main/java/de/metas/document/dimension/Dimension.java

```diff
@@ -194,6 +204,8 @@ else if (collection.size() == 1)
 				.user2_ID(singleElementOrZero(user2_ID))
 				.userElement1Id(singleElementOrZero(userElement1Id))
 				.userElement2Id(singleElementOrZero(userElement2Id))
+				.userElementNumber1(CollectionUtils.singleElementOrDefault(userElementNumber1, BigDecimal.ZERO))
+				.userElementNumber2(CollectionUtils.singleElementOrDefault(userElementNumber2, BigDecimal.ZERO))
```

Comment:
why not singleElementOrNull ?
(talking about UserElementNumber1 and UserElementNumber2)